kloppy/tests/test_tracab.py

```diff
@@ -18,19 +18,122 @@
 from kloppy import tracab
 
 
-class TestTracabTracking:
-    @pytest.fixture
-    def meta_data(self, base_dir) -> str:
-        return base_dir / "files/tracab_meta.xml"
+@pytest.fixture(scope="session")
+def json_meta_data(base_dir: Path) -> Path:
+    return base_dir / "files" / "tracab_meta.json"
 
-    @pytest.fixture
-    def raw_data(self, base_dir) -> str:
-        return base_dir / "files/tracab_raw.dat"
 
-    def test_correct_deserialization(self, meta_data: Path, raw_data: Path):
+@pytest.fixture(scope="session")
+def json_raw_data(base_dir: Path) -> Path:
+    return base_dir / "files" / "tracab_raw.json"
+
+
+@pytest.fixture(scope="session")
+def xml_meta_data(base_dir: Path) -> Path:
+    return base_dir / "files" / "tracab_meta.xml"
+
+
+@pytest.fixture(scope="session")
+def dat_raw_data(base_dir: Path) -> Path:
+    return base_dir / "files" / "tracab_raw.dat"
+
+
+def test_correct_auto_recognize_deserialization(
```

Comment:
I think testing whether the `identify_deserializer` function returns the correct deserializer class would be better.